src/api.v2/model/Plot.js

```diff
@@ -0,0 +1,56 @@
+const config = require('../../config');
+
+const BasicModel = require('./BasicModel');
+const sqlClient = require('../../sql/sqlClient');
+
+const validateRequest = require('../../utils/schema-validator');
+
+const tableNames = require('./tableNames');
+const getObject = require('../helpers/s3/getObject');
+
+const selectableProps = [
+  'id',
+  'experiment_id',
+  'config',
+  's3_data_key',
+];
+
+class Plot extends BasicModel {
+  constructor(sql = sqlClient.get()) {
+    super(sql, tableNames.PLOT, selectableProps);
+  }
+
+  async getConfig(experimentId, plotUuid) {
+    const {
+      s3DataKey,
+      config: plotConfig,
+    } = await this.findOne({ id: plotUuid, experiment_id: experimentId });
+
+    const result = { config: plotConfig };
+
+
+    if (s3DataKey) {
+      const bucketName = `plots-tables-${config.clusterEnv}`;
```

Comment:
this bucket name should be in a config, not hardcoded here. Can you move it to the default config?

Reply:
Turns out wie have `/helpers/s3/bucketNames.js`. I'll add it there.